Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -362,8 +362,8 @@ class CheckmarkSetting: Setting {
 
     override func onConfigureCell(_ cell: UITableViewCell) {
         super.onConfigureCell(cell)
-        cell.accessoryType = isEnabled() ? .checkmark : .none
-        cell.selectionStyle = .none
+        cell.accessoryType = .checkmark
+        cell.tintColor = isEnabled() ? SettingsUX.TableViewRowActionAccessoryColor : UIColor.white
```

Comment:
Bug 1421718 - Swap default keyboard appearance to dark in Private Browsing mode.

When the checkbox is disabled, the tableview is reloaded without the accessory view, and the text is re-wrapped because the content view takes the whole width of the cell.

This fix makes hides the checkbox by making it match the background color, avoiding re-wrapping the label text.

Reply:
That bug doesn't seem to mention this so I added a bug https://bugzilla.mozilla.org/show_bug.cgi?id=1458903